### PR TITLE
SWTCH-1247 specify format

### PR DESCRIPTION
**Pull request by dawidgil**

### CODE REVIEW

File: src/app/routes/registration/enrolment-form/enrolment-form.component.spec.ts [hunk 1]
```diff
@@ -119,6 +119,62 @@ describe('EnrolmentFormComponent', () => {
...
+
+      const {offChain, onChain} = getSelectors(hostDebug);
+
+      expect(offChain.nativeElement.checked).toBeTruthy('off chain should be selected');
+      expect(offChain.nativeElement.disabled).toBeTruthy('off chain is not disabled');
+      expect(onChain.nativeElement.checked).toBeFalsy('on chain should be deselected');
+      expect(onChain.nativeElement.disabled).toBeTruthy('on chain is not disabled');
+    });
+
+    it('should disable checkboxes when offChain and onChain are predefined and set to true', () => {
```
Comment (jrhender) [inline src/app/routes/registration/enrolment-form/enrolment-form.component.spec.ts:151]: Should this say `enable`?
```suggestion
    it('should enable checkboxes when offChain and onChain are predefined and set to true', () => {
```

File: src/app/routes/registration/enrolment-form/enrolment-form.component.spec.ts [hunk 2]
```diff
@@ -119,6 +119,62 @@ describe('EnrolmentFormComponent', () => {
...
+      component.fieldList = [];
+      component.predefinedRegTypes = {offChain: true, onChain: true};
+
+      fixture.detectChanges();
+
+      const {offChain, onChain} = getSelectors(hostDebug);
+
+      expect(offChain.nativeElement.checked).toBeTruthy('off chain should be selected');
+      expect(offChain.nativeElement.disabled).toBeTruthy('off chain is disabled');
```
Comment (jrhender) [inline src/app/routes/registration/enrolment-form/enrolment-form.component.spec.ts:160]: These messages are the error messages, right?
```suggestion
      expect(offChain.nativeElement.disabled).toBeTruthy('off chain is not disabled');
```

**Review by jrhender — COMMENTED** [review]
Just some comments about the messages

File: src/app/routes/registration/enrolment-form/enrolment-form.component.spec.ts (hunk 1 above)
Comment (dawidgil) [reply src/app/routes/registration/enrolment-form/enrolment-form.component.spec.ts:151]: No, what i understood checkboxes should be disabled when are predefined by url. So if we have in url onchain=true then checkbox for on-chain should be disabled but checked. Checkbox for off-chain should be also disabled but unchecked (disabled means when i click on it, nothing happens)

File: src/app/routes/registration/enrolment-form/enrolment-form.component.spec.ts (hunk 2 above)
Comment (dawidgil) [reply src/app/routes/registration/enrolment-form/enrolment-form.component.spec.ts:160]: yes, useful for debugging.

File: src/app/routes/registration/enrolment-form/enrolment-form.component.spec.ts (hunk 1 above)
Comment (jrhender) [reply src/app/routes/registration/enrolment-form/enrolment-form.component.spec.ts:151]: Oh yes, you are right 👍

**Comment by dawidgil** [comment]
Adjusted error information in unit tests. I used one approach which is `checkbox should be/not be disabled` to remove confusion. Hope that now it looks good :)

**Review by jrhender — APPROVED** [review]
Looks great, thanks!

**Pull request merged**